.github/FUNDING.yml

```diff
@@ -1,5 +1,7 @@
+---
 # These are supported funding model platforms
 
+# yamllint disable rule:comments rule:line-length
```

Comment:
You could instead just drop the comments and this wouldn't be necessary.

Reply:
That's true, but I don't consider the file meaningful to this project. I assume it is based on a GitHub template and the comment might still be useful to someone in the future.

Reply:
It's just usually better not to have any ignores. Whenever you have an ignore, it should be accompanied with a code comment explaining why it's there and why can't it just be fixed.

Reply:
I guess you're right! Comments are gone. Future someone should be able to look this up in git history or GitHub docs.